tracing/config/tracing.go

```diff
@@ -98,39 +99,29 @@ func NewTracingConfigFromMap(cfgMap map[string]string) (*Config, error) {
 		}
 	}
 
-	if endpoint, ok := cfgMap[zipkinEndpointKey]; !ok && tc.Backend == Zipkin {
+	if err := cm.Parse(cfgMap,
```

Comment:
Why don't you parse `backendKey` and `enable` here in a local variable and then you move all the validation algorithm below?

Reply:
Lemme see how that feels like.

Reply:
Ah: It's special cased logic that's in an either/or fashion. I can't properly express that in the logic here so I figured I'd just keep it as-is.